script/backport-patchset.py

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env python
+
+import os
+import re
+import sys
+import urllib2
+
+SOURCE_ROOT = os.path.abspath(os.path.dirname(os.path.dirname(__file__)))
+PATCHES_DIR = os.path.join(SOURCE_ROOT, 'patches')
+
+def main():
+  if len(sys.argv) < 3:
+    print("Usage: backport-patchset.py [code-review-id] [patch-name]")
+    return 1
+  codereview_url = "https://codereview.chromium.org/" + sys.argv[1]
```

Comment:
_(trivial)_ `sys.argv[1]` is referred to three times. It would be clearer to use a local instead, e.g.

`review_id = sys.argv[1]`